index.js

```diff
@@ -1,31 +1,45 @@
-/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
-   Complete the below for code reviewers' convenience:
-
-   Code repository: _put repo URL here_
-   Web app: _put project's github pages URL here_
-   */
-
-// ======== OBJECTS DEFINITIONS ========
-// Define your objects here
-
-
-// ======== OUTPUT ========
-/* Use print(message) for output.
-   Default tag for message is <pre>. Use print(message,'div') to change containing element tag.
-
-   Message can contain HTML markup. You may also tweak index.html and/or styles.css.
-   However, please, REFRAIN from improving visuals at least until your code is reviewed
-   so code reviewers might focus on a single file that is index.js.
-   */
-
-/* Print examples:
-   print('ABC');
-   print('<strong>ABC</strong>');
-   print('<strong>ABC</strong>', 'div');
-
-   print('human; John; male; 2; 2; Hello world!; Rex, Tom, Jenny');
-   print('human; <strong>John</strong>; male; 2; 2; <em>Hello world!</em>; Rex, Tom, Jenny');
-   print('human; <strong>John</strong>; male; 2; 2; <em>Hello world!</em>; Rex, Tom, Jenny', 'div');
-   */
-
-
+class Parent {
+  constructor(name, gender, kind, hands, legs, say) {
+    this.name = name
+    this.gender = gender
+    this.kind = kind
+    this.hands = hands
+    this.legs = legs
+    this.say = say
+  }
+  printAll() {
+    return Object.keys(this)
```

Comment:
Here is your bonus: rename this method to `toString`. Now you will able to `print(obj)` in line 44.

But here is the problem to solve next: the order of properties returned by `#Object.keys|values|entries` is not guaranteed. We need different approach. Obviously the one is to list properties explicitly.